pkg/endpoint/identifiers.go

```diff
@@ -11,9 +11,12 @@ import (
 
 // GetContainerName returns the name of the container for the endpoint.
 func (e *Endpoint) GetContainerName() string {
-	e.unconditionalRLock()
-	defer e.runlock()
-	return e.containerName
+	// const after creation
```

Comment:
This comment seems slightly off as this is not set on the initial endpoint creation when using the docker plugin?

Reply:
Oh, you're right, it's write-once. I'll update the comment.